wikidp/static/js/item_contribute.js

```diff
@@ -20,26 +20,31 @@ const initializeStatementPropertySelector = () => {
   );
 };
 
-
```

Comment:
The majority of the diff in this file  is cleaning up the code styling to better conform to modern ESLint conventions. (indentation, camelCase, const, etc)

The functional changes are abstracting qualifier code to reuse for references